Source/Modules/php.cxx

```diff
@@ -441,6 +457,9 @@ class PHP : public Language {
     Printf(s_header, "}\n");
     Printf(s_header, "#endif\n\n");
 
+    Printf(s_header,"#ifdef __GNUG__\n#define SWIG_remove(zv) delete zv\n");
+    Printf(s_header,"#else\n#define SWIG_remove(zv) free(zv)\n#endif\n\n");
```

Comment:
This is because, the pointer in struct needs to be freed. And free is used in C and delete in C++. delete calls the destructor.